sinks/ssfmetrics/README.md

```diff
@@ -0,0 +1,24 @@
+# SSFMetrics Sink
+
+This sink creates timer metrics from SSF indicator spans.
+
+# Status
+
+**This sink is experimental**. Names and tags are likely to change.
```

Comment:
I think it's fair to call this one stable, since we're enabling it unconditionally/unconfigurably (-: